### DISCUSSION TITLE
guide/access/ssh

### DISCUSSION THREAD

**Discussion by giscus** [Eeems-Org/remarkable.guide#81] [General]
# guide/access/ssh

If you don’t write down your password and lose access to the UI, you will be unable to access your device. An Emergency Recovery will be required. It is also recommended to setup a ssh-key instead ...

https://remarkable.guide/guide/access/ssh.html

**Comment by nycterent** [comment]
had to use -t rsa to generate key that was accepted by my remarkable 2

**Reply by Eeems** [reply]
What OS version is your device running?

**Comment by nycterent** [comment]
it used to be 2.XX, but after the upgrade to 3.14.1.9 everything is working
as expected

m
